python/django/userdata/views.py

```diff
@@ -72,8 +77,22 @@ def post(self, request):
         serializer = self.serializer_class(data=request.DATA)
         if serializer.is_valid():
             user = serializer.object['user']
+
             token, created = Token.objects.get_or_create(user=user)
-            return Response({'token': token.key, 'user': token.user_id})
+
+            # check if first login
+            userObject = OTIUser.objects.get(username=user)
+
+             # check if this is the first time user has logged in
+            first_login = False
+            if userObject and userObject.last_login < LOGIN_FLAG_DATE:
```

Comment:
This is always ok? What does last_login default to when a new user object is created?